test.e2e/spec/services.spec.js

```diff
@@ -1102,7 +1102,7 @@ define(['angular', 'given', 'util'], function(angular, given, util) {
           // Type "number" was normalized to "Number"
           price: { type: 'Number' },
           // auto-injected id property
-          id: { id: 1, generated: true, type: 'Number' },
+          id: { id: 1, generated: true, type: 'Number', updateOnly: true },
```

Comment:
I have mixed feelings about this change. Sure, it will fix the problem once all packages affected by `updateOnly` are published to npmjs, but until that happens, people may start seeing build failures.

I am proposing to rework this test and make it more resilient against possible changes in model definitions. I am thinking about a small change - replace `eql` operator with something like `.contains` or `.have.properties` (I never remember what is the right [chai](http://chaijs.com/api/bdd/) method to use). With this change in place, the test should be passing against both the currently-release loopback/juggler version and the upcoming one.

Thoughts?